packages/framework/src/Component/Cron/CronFacade.php

```diff
@@ -55,17 +55,42 @@ public function runScheduledModules()
         $cronModuleExecutor = new CronModuleExecutor(self::TIMEOUT_SECONDS);
 
         $cronModuleConfigs = $this->cronConfig->getAllCronModuleConfigs();
+
         $scheduledCronModuleConfigs = $this->cronModuleFacade->getOnlyScheduledCronModuleConfigs($cronModuleConfigs);
         $this->runModules($cronModuleExecutor, $scheduledCronModuleConfigs);
     }
 
+    /**
+     * @param string $instanceName
+     */
+    public function runScheduledModulesForInstance(string $instanceName): void
+    {
+        $cronModuleExecutor = new CronModuleExecutor(self::TIMEOUT_SECONDS);
+
+        $cronModuleConfigs = $this->cronConfig->getCronModuleConfigsForInstance($instanceName);
+
+        $scheduledCronModuleConfigs = $this->cronModuleFacade->getOnlyScheduledCronModuleConfigs($cronModuleConfigs);
+        $this->runModulesForInstance($cronModuleExecutor, $scheduledCronModuleConfigs, $instanceName);
+    }
+
     /**
      * @param \Shopsys\FrameworkBundle\Component\Cron\CronModuleExecutor $cronModuleExecutor
      * @param \Shopsys\FrameworkBundle\Component\Cron\Config\CronModuleConfig[] $cronModuleConfigs
+     * @deprecated Use `runModulesForInstance` instead
      */
     protected function runModules(CronModuleExecutor $cronModuleExecutor, array $cronModuleConfigs)
     {
-        $this->logger->addInfo('====== Start of cron ======');
+        $this->runModulesForInstance($cronModuleExecutor, $cronModuleConfigs, CronModuleConfig::DEFAULT_INSTANCE_NAME);
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Cron\CronModuleExecutor $cronModuleExecutor
+     * @param array $cronModuleConfigs
+     * @param string $instanceName
+     */
+    protected function runModulesForInstance(CronModuleExecutor $cronModuleExecutor, array $cronModuleConfigs, string $instanceName): void
+    {
+        $this->logger->addInfo("====== Start of cron instance {$instanceName} ======");
```

Comment:
I think we do not use this way of variables outputting, I would prefer using `sprintf` :innocent:

Reply:
Do we have this recommendation documented somewhere? It could be nice to have then.

Reply:
Hm, good question, it should be probably here - https://github.com/shopsys/shopsys/blob/master/docs/contributing/coding-standards.md

Reply:
or even better, there should be sniff for that :slightly_smiling_face:

Reply:
https://github.com/shopsys/shopsys/issues/839

Reply:
we should have it registered PHP_CodeSniffer\Standards\Squiz\Sniffs\Strings\DoubleQuoteUsageSniff